docs/src/content/docs/fr/reference/configuration.mdx

```diff
@@ -516,6 +514,13 @@ Définit le délimiteur entre le titre de la page et le titre du site dans la ba
 Par défaut, chaque page a une balise `<title>` contenant `Titre de la page | Titre du site`.
 Par example, cette page a pour titre « Référence de configuration » et ce site a pour titre « Starlight », donc la balise `<title>` de cette page contient « Référence de configuration | Starlight ».
 
+### `disable404Route`
+
+**Type :** `boolean`  
+**Par défaut :** `'false'`
+
+Désactive l'injection de la [page 404](https://docs.astro.build/fr/core-concepts/astro-pages/#page-derreur-404-personnalis%C3%A9e) par défaut de Starlight. Pour utiliser une route personnalisée `src/pages/404.astro` dans votre projet, définissez cette option à `false`.
```

Comment:
```suggestion
Désactive l'injection de la [page 404](https://docs.astro.build/fr/core-concepts/astro-pages/#page-derreur-404-personnalis%C3%A9e) par défaut de Starlight. Pour utiliser une route personnalisée `src/pages/404.astro` dans votre projet, définissez cette option à `true`.
```

One *last* change to be updated with the changes of https://github.com/withastro/starlight/pull/1431